app/controllers/api/users.php

```diff
@@ -1420,6 +1422,235 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             ->dynamic($target, Response::MODEL_TARGET);
     });
 
+App::post('/v1/users/:userId/sessions')
+    ->desc('Create session')
+    ->groups(['api', 'users'])
+    ->label('event', 'users.[userId].sessions.create')
```

Comment:
This needs to be `users.[userId].sessions.[sessionId].create`

Reply:
Updated

Reply:
Done